Blockzilla/en.lproj/InfoPlist.strings

```diff
@@ -1,4 +1,7 @@
 /* (No Commment) */
```

Comment:
This needs to go. Sorry if the wiki did not make it clear. But no `CF*` keys should be in these files.